src/components/accounts/SetupImplicit.js

```diff
@@ -119,7 +119,7 @@ class SetupImplicit extends Component {
         const { accountFunded } = this.state;
 
         if (!accountFunded) {
-            const account = await dispatch(getAccountBasic(implicitAccountId));
+            const account = dispatch(getAccountBasic(implicitAccountId));
```

Comment:
Since `getAccountBasic()` is now _entirely synchronous_, and has no error handling metadata associated with the action, I don't think we get _any_ benefit from `dispatch()`ing it at all.  A brief search for this shows me that this is the _only place in the codebase_ that uses this action --  _Everywhere_ else just calls `wallet.getAccountBasic()` directly). Let's remove the action entirely and just call `wallet.getAccountBasic()` directly here for consistency, instead of having this go through Redux unnecessarily.

Reply:
Done.